gcc/testsuite/gcc.dg/tree-ssa/ssa-store-ccp-2.c

```diff
@@ -1,5 +1,5 @@
 /* { dg-do compile } */
-/* { dg-options "-O2 -fdump-tree-optimized" } */
+/* { dg-options "-O2 -fdump-tree-optimized -mno-sdata" } */
```

Comment:
-O2 and dump tree should be not arch dependent. So like this?
```
/* { dg-options "-O2 -fdump-tree-optimized" { target arc*-*-* } } */
/* { dg-options "-O2 -fdump-tree-optimized -mno-sdata" { target arc*-*-* } } */
```
I don't really know.